### PR TITLE
[1846] Use new template for renewal letter

### PR DESCRIPTION
**Pull request by d-a-v-e**
https://eaflood.atlassian.net/browse/RUBY-1846

### CODE REVIEW

File: app/presenters/notify_renewal_letter_presenter.rb [hunk 1]
```diff
@@ -6,7 +6,11 @@ class NotifyRenewalLetterPresenter < BasePresenter
 
   def expiry_date
     # Currently you can only add exemptions when you register, so we can assume they expire at the same time
-    registration_exemptions.first.expires_on.to_formatted_s(:day_month_year)
+    first_exemption.expires_on.to_formatted_s(:day_month_year)
+  end
+
+  def renewal_window_start_date
+    (first_exemption.expires_on - renewal_window_after_expiry_in_days).to_formatted_s(:day_month_year)
```
Comment (timstone) [inline app/presenters/notify_renewal_letter_presenter.rb]: don't you mean `renewal_window_before_expiry_in_days` ?

File: config/initializers/waste_exemptions_engine.rb [hunk 2]
```diff
@@ -37,7 +37,7 @@
   configuration.use_last_email_cache = ENV["USE_LAST_EMAIL_CACHE"] || "false"
 
   # Renewing config
-  configuration.renewal_window_before_expiry_in_days = ENV["RENEWAL_WINDOW_BEFORE_EXPIRY_IN_DAYS"] || 28
+  configuration.renewal_window_before_expiry_in_days = ENV["RENEWAL_WINDOW_BEFORE_EXPIRY_IN_DAYS"] || 35
```
Comment (timstone) [inline config/initializers/waste_exemptions_engine.rb]: We don't want to change the renewal window, it should be 28 days

File: config/application.rb [hunk 3]
```diff
@@ -67,7 +67,7 @@ class Application < Rails::Application
     config.epr_export_filename = ENV["EPR_DAILY_REPORT_FILE_NAME"] || "waste_exemptions_epr_daily_full"
     config.export_batch_size = ENV["EXPORT_SERVICE_BATCH_SIZE"] || 1000
 
-    config.ad_letters_exports_expires_in = ENV["AD_LETTERS_EXPORT_EXPIRES_IN"] || 35
+    config.ad_letters_exports_expires_in = ENV["AD_LETTERS_EXPORT_EXPIRES_IN"] || 33
```
Comment (timstone) [inline config/application.rb]: I think the suggestion was to make this a dynamic number as there could be 28 days in the month, so one month + 3 days.

**Review by timstone — COMMENTED** [review]
I won't appove PR's but that makes more sense to me now 👍

File: spec/services/notify_renewal_letter_service_spec.rb [hunk 4]
```diff
@@ -3,24 +3,23 @@
...
+  describe ".run" do
     let(:registration) { create(:registration) }
-    let(:service) do
-      NotifyRenewalLetterService.run(registration: registration)
-    end
+    before { registration.registration_exemptions.update_all(expires_on: Date.new(2030, 7, 1)) }
 
     it "sends a letter" do
-      VCR.use_cassette("notify_renewal_letter") do
+      VCR.use_cassette("notify_renewal_letter", record: :new_episodes) do
```
Comment (PaulDoyle-EA) [inline spec/services/notify_renewal_letter_service_spec.rb]: Minor point and not a blocker, but we shouldn't have to specify in an individual test that VCR can overwrite an existing cassette. The configuration might need some attention.

**Review by PaulDoyle-EA — DISMISSED** [review]

File: spec/services/notify_renewal_letter_service_spec.rb (hunk 4 above)
Comment (d-a-v-e) [reply spec/services/notify_renewal_letter_service_spec.rb]: ah yeah, this is no longer needed.

**Review by PaulDoyle-EA — APPROVED** [review]

**Pull request merged**